README.md

```diff
@@ -1,5 +1,18 @@
 # codevid19.com
 
+## Running with Docker
+
+Assuming Docker and docker-compose are installed:
+```{bash}
+make start
+```
+Then from within the container you can run:
+```
+./serve.sh
```

Comment:
We can probably just move this serve command right into the `Makefile`. Then it can be used from inside and out of the dev container.